### PR TITLE
PHPC-1792: Change all copyrights to be <year>-present

### PR DESCRIPTION
**Pull request by agolin95**
Some of these were in the form of `Copyright <year> MongoDB, Inc.` as opposed to `Copyright <year>=<year> MongoDB, Inc.` - I changed those as well assuming we would want equivalence across files, but please lmk if that was an incorrect assumption!

I also opted for "Present" instead of "present" (template updated accordingly) - please lmk if you aren't a fan of that!

### CODE REVIEW

**Comment by jmikola** [comment]
I'm in favor of lowercase "present". That was in the original template from legal and I believe it is what is used in other projects (server for sure).

As for changing `<year>` to `<year>-present`, I think that's correct. Before we started using "present", files that were introduced only received the current year. If they were updated in a later year, that line was then updated with a range. So all copyright lines in the driver should now be in the form "<first year introduced>-present". 

> template updated accordingly

FYI, `template.rc` is just a file for the Windows DLL produced by PECL. It's serves no purpose for the driver itself.

**Comment by agolin95** [comment]
@jmikola thank you for the review and sorry for the delay! Pushed a commit that changes titlecase "Present" to lowercase "present"

File: template.rc [hunk 1]
```diff
@@ -38,7 +38,7 @@ BEGIN
             VALUE "FileDescription", FILE_DESCRIPTION
             VALUE "FileVersion", PHP_MONGODB_VERSION
             VALUE "InternalName", FILE_NAME
-            VALUE "LegalCopyright", "Copyright � 2014-present MongoDB, Inc."
+            VALUE "LegalCopyright", "Copyright � 2014-present MongoDB, Inc."
```
Comment (alcaeus) [inline template.rc:41]: This looks jumbled, I think we can revert the change to this file.

**Review by alcaeus — APPROVED** [review]
LGTM besides the template.rc change

File: template.rc (hunk 1 above)
Comment (alcaeus) [reply template.rc:41]:
```suggestion
            VALUE "LegalCopyright", "Copyright © 2014-present MongoDB, Inc."
```

**Review by alcaeus — CHANGES_REQUESTED** [review]

**Review by alcaeus — APPROVED** [review]

**Pull request merged**